src/components/black-friday/payment/PriceCard.tsx

```diff
@@ -43,14 +43,14 @@ export default function PriceCard({
   };
 
   const convertedPrice = useMemo(() => {
-    if (country !== '€') {
+    if (currency !== '€') {
       const splitPrice = price.toString().split('.');
       const checkDecimalPrice = splitPrice[1] >= '50' ? 0.99 : 0.49;
```

Comment:
This smells fishy:

- splitPrice[1] can be easily null or undefined
- You are checking if an string is major or equal to an string (which sounds really weird)
- I don't understand this logic
- You better move this logic to a centralized file like PricingService or something like that

There are already solutions around this kind of things:

https://www.samanthaming.com/tidbits/30-how-to-format-currency-in-es6/
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat